content/about.md

```diff
@@ -0,0 +1,27 @@
+---
+title: About
+---
+
+This website intends to pool resources and information on "transformational spaces". 
+It is part of a wider project at Life Itself, in conjunction with Fetzer, Ekskaret, and Commonweal to gather data on and eventually build a field for transformational spaces. Our focus is on an initial "survey of the field", and to map stakeholders and programs and start forging connections and awareness.  Now, we are on programs for youth and young adults (15-25 years old) in the US and Europe, and we will be exploring what it means to be a "transformational space", profile relevant organizations, looking at their programs, approach and philosophy.
```

Comment:
Now we are **focusing** on programs for youth ...

we will be exploring... **profiling**...